### PR TITLE
Adaption to aldryn-boilerplates

### PR DESCRIPTION
**Pull request by FinalAngel**
* multi-boilerplate support
  new requirement: aldryn-boilerplates (needs configuration)

### CODE REVIEW

File: aldryn_forms/boilerplates/legacy/templates/aldryn_forms/base_form.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{# This is a very nasty hack to redirect the user #}
+{% if form_success_url %}
+<script type="text/javascript">
+    window.location.replace("{{ form_success_url }}");
```
Comment (jmelett) [inline aldryn_forms/boilerplates/legacy/templates/aldryn_forms/base_form.html]: Not sure if I understand correctly - shouldn't we do this in the backend?

Comment (FinalAngel) [inline aldryn_forms/boilerplates/legacy/templates/aldryn_forms/base_form.html]: that is correct, but apparently not possible due to some response handle issues from the backend (cms related) /cc @czpython

Comment (czpython) [inline aldryn_forms/boilerplates/legacy/templates/aldryn_forms/base_form.html]: @jmelett indeed we should, sadly this is one of the limitations we have when working with plugins. They can't return a response (in this case a redirect).

**Comment by jmelett** [comment]
:+1:

**Pull request merged**